src/silx/gui/dialog/ColormapDialog.py

```diff
@@ -66,6 +66,7 @@
 import logging
 
 import numpy
+from future import __annotations__
```

Comment:
This needs to be the very first of executable line of code right after the module docstring.

And it's: `from __future__ import annotations` ;)

Reply:
nice one ^^